src/test/java/it/clients/symphony/api/HealthcheckClientTest.java

```diff
@@ -43,4 +46,27 @@ public void performHealthCheckSuccess() {
     assertEquals(true, response.getPodConnectivity());
   }
 
+  @Test
+  public void performHealthCheckSuccessWithUnknownField() {
+    stubFor(get(urlEqualTo(AgentConstants.HEALTHCHECK))
```

Comment:
Could you consider using utility method for mocking HTTP calls : https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/src/test/java/it/commons/BotTest.java#L48 ?

Reply:
Thanks, I made the changes.